README.md

```diff
@@ -1 +1,53 @@
-# stephen-tribute-page
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+tribute page for the famous scientist Stephen Hawking
+
+## Getting Started
+
+step 1:
+Clone this repo locally using git in the command line and the following command
+
+git clone https://github.com/hammaazarok/hello-microverse.git
+
+Alternatively, you can just download the complete zip file and extract the folder in your directory
+
+Step 2:
+Copy All Project files and run using VSCODE. HTML can be viewed in browser.
+
+### Prerequisites
+
+Any html and css code editor like Vscode or Vim.
+
+## Built With
+
+- Html
+- CSS
+
+## Authors
+
+👤 **Hamma Azarok**
+
+- GitHub: [@hammaazarok](https://github.com/hammaazarok)
+
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Miroverse Inc.
+- Qalb-E-Al
+
```

Comment:
[OPTIONAL] - [ ] you can add the Acknowledge to the freeCodeCamp site as you used their resources while your project.